.github/workflows/tests.yaml

```diff
@@ -0,0 +1,29 @@
+name: tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  translations-check:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [18, 17, 16, 15, 14, 12, 10, 10.13]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Tests
+        run: npm run pretest && npm run tests-only
```

Comment:
linting should not be run on every matrix job, it should be run separately and only once.